src/webparts/controlsTest/ControlsTestWebPart.ts

```diff
@@ -1,30 +1,31 @@
-import * as React from "react";
-import * as ReactDom from "react-dom";
+import * as React from 'react';
```

Comment:
Can you try to revert the formatting changes?

Reply:
done